core/src/main/java/io/undertow/client/UndertowHttp2Client.java

```diff
@@ -0,0 +1,173 @@
+package io.undertow.client;
+
+import io.undertow.connector.ByteBufferPool;
+import org.xnio.FutureResult;
+import org.xnio.IoFuture;
+import org.xnio.OptionMap;
+import org.xnio.XnioIoThread;
+import org.xnio.XnioWorker;
+import org.xnio.ssl.XnioSsl;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URI;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ServiceLoader;
+
+/**
+ * Undertow http/2 client class. This class loads {@link ClientProvider} implementations, and uses them to
+ * create connections to a target.
+ *
+ * @author Steve Hu
+ */
+public class UndertowHttp2Client {
```

Comment:
What is the purpose of this class? The standard HTTP client can handle HTTP/2 connections?